pkg/api/types.go

```diff
@@ -43,6 +43,9 @@ type DeschedulerPolicy struct {
 
 	// MaxNoOfPodsToEvictPerNode restricts maximum of pods to be evicted per node.
 	MaxNoOfPodsToEvictPerNode *int
+
+	// MaxNoOfPodsToEvictPerNamespace restricts maximum of pods to be evicted per namespace.
+	MaxNoOfPodsToEvictPerNamespace *int
```

Comment:
`MaxNoOfPodsToEvictPerNamespace *uint` since negative values are not valid.

Reply:
Fix done~